crates/stackable-webhook/src/servers/conversion.rs

```diff
@@ -1,14 +1,61 @@
 use std::fmt::Debug;
 
-use axum::{Json, Router, extract::State, routing::post};
+use axum::{Json, Router, routing::post};
+use k8s_openapi::{
+    ByteString,
+    apiextensions_apiserver::pkg::apis::apiextensions::v1::{
+        CustomResourceConversion, CustomResourceDefinition, ServiceReference, WebhookClientConfig,
+        WebhookConversion,
+    },
+};
 // Re-export this type because users of the conversion webhook server require
 // this type to write the handler function. Instead of importing this type from
 // kube directly, consumers can use this type instead. This also eliminates
 // keeping the kube dependency version in sync between here and the operator.
 pub use kube::core::conversion::ConversionReview;
+use kube::{
+    Api, Client, ResourceExt,
+    api::{Patch, PatchParams},
+};
+use snafu::{OptionExt, ResultExt, Snafu};
+use stackable_operator::cli::OperatorEnvironmentOptions;
+use tokio::{sync::mpsc, try_join};
 use tracing::instrument;
+use x509_cert::{
+    Certificate,
+    der::{EncodePem, pem::LineEnding},
+};
```

Comment:
note: Please combine these imports with the others above.

Reply:
hmm, what imports exactly can be combined? rustfmt groups them like this

Reply:
Yeah this is sub-optimal formatting sadly. You need to manually move them to the top imports above line 11.